### ISSUE TITLE
There is no prompt for a function that does not exist

### ISSUE THREAD

**Issue by wangxudong123** [intellij-rust/intellij-rust#8525]
## Environment

* **IntelliJ Rust plugin version:** 0.4.162.4321-213
* **Rust toolchain version:** I don't know
* **IDE name and version:** CLion CL-213.5744.254
* **Operating system:** macos 12.1

## Problem description
I wrote a function, but I didn't implement it. Rust plug-in has no error prompt. I hope it has wavy lines

## Steps to reproduce
```rust

fn main() {
   println!("hello");

    //didn't implement
    //I hope it has wavy lines
    Test();
    
    return
}

```

**Comment by vlad20012** [comment]
Actually, the plugin can show you this error via embedded inspections but it is disabled by default in some cases not to show possible false positives. But if you want to try, disable `Ignore unresolved references without quick fix` option of `Rust | Unresolved reference` inspection in `Settings / Preferences | Editor | Inspections` settings.
Eventually, they will be enabled by default in all cases.

Also, the plugin has a feature called `External Linter`. It runs `cargo check` or `cargo clippy` in the background and highlights your code in the editor in accordance with the result (vscode uses the same approach). You can enable `Run external linter to analyze code on the fly` option in `Settings / Preferences | Languages & Frameworks | Rust | Cargo`. See https://plugins.jetbrains.com/plugin/8182-rust/docs/rust-code-analysis.html#external-linters for more details

**Comment by wangxudong123** [comment]
i disable `Ignore unresolved references without quick fix` , but it turns correct hints into wrong
